integration-tests/tests/test_cases.py

```diff
@@ -0,0 +1,89 @@
+import json
+import requests
+from conftest import option, user_1, user_2
```

Comment:
Unused imports 
```suggestion
from conftest import option
```

Reply:
they are actually used in transaction data, https://github.com/status-im/status-go/blob/45be54008ed665f4441d5c86ff4b67c4a7b0b631/integration-tests/tests/test_cases.py#L63